dasbus/connection.py

```diff
@@ -286,14 +286,22 @@ def publish_object(self, object_path, obj,
         )
         object_handler.connect_object()
 
-        self._registrations.append(object_handler.disconnect_object)
+        self._registrations[object_path] = object_handler.disconnect_object
+
+    def unregister(self, object_path):
```

Comment:
I would prefer to have the `unregister_service` and `unpublish_object` methods to make it more future-proof. Then the logging messages can match their counterparts.

Reply:
Okay sure... hows that?

Reply:
It looks good. Could you also rename the `unregister` method  to `_unregister` to make it private, please? That would be perfect.